app/controllers/hiring_staff/vacancies/application_controller.rb

```diff
@@ -18,6 +18,10 @@ def set_vacancy
     @vacancy = params[:job_id] ? current_school.vacancies.find(params[:job_id]) : nil
   end
 
+  def current_step
```

Comment:
This method is also defined in the new module HiringStaff::JobCreationHelper. Can we use that one here?

Reply:
We could, but I don't know if that means we should